adapters/feishu/src/modulator.ts

```diff
@@ -2,7 +2,14 @@ import { Modulator, segment } from '@satorijs/core'
 import { FeishuBot } from './bot'
 
 export class FeishuModulator extends Modulator<FeishuBot> {
-  private buffer: any
+  private mode: 'default' | 'figure' = 'default'
+  private content = ''
+  private addition = {
+    file: null as {
+      id: string
+      type: FeishuBot.Message.FileContents['msg_type']
+    } | null,
```

Comment:
Personally I don't like `null`, but prefer `undefined` to refer to a nil value.

Reply:
Actually you can omit this `| null` because we do not use SNC.

Reply:
True, but I am also refering the `file: null as {...}` part, which can be just `undefined` in this case.